src/ore/src/cast.rs

```diff
@@ -41,14 +41,14 @@ macro_rules! cast_from {
 #[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
 cast_from!(u32, usize);
 
-#[cfg(target_pointer_width = "64")]
-cast_from!(i32, usize);
-
 #[cfg(target_pointer_width = "64")]
 cast_from!(u64, usize);
 
 cast_from!(usize, u64);
 
+#[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
+cast_from!(i32, isize);
```

Comment:
maybe a dumb question but is this being used in any of the other changes in this PR. looked twice but i didnt see any i32 -> isize conversions maybe I'm missing something

Reply:
You're not! I left it for completeness's sake. When these have been added piecemeal incorrect ones have slipped in. :)